src/main/java/mate/academy/spring/dto/MovieSessionRequestDto.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.spring.dto;
+
+import java.time.LocalDateTime;
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.model.Movie;
+
+public class MovieSessionRequestDto {
+    private Movie movie;
+    private CinemaHall cinemaHall;
```

Comment:
You should't pass objects of Movie and CinemaHall . You need to pass only their ids.
```suggestion
    private Long movieId;
    private Long cinemaHallId;
```